recipes/vtk/all/conanfile.py

```diff
@@ -0,0 +1,176 @@
+import os
+import re
+import shutil
+
+from fnmatch import fnmatch
+from conans import ConanFile, CMake, tools
+
+class VTKConan(ConanFile):
+    name = "vtk"
+    version = "9.0.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://vtk.org/"
+    license = "BSD license"
+    description = "Visualization Toolkit by Kitware"
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    source_subfolder = "source_subfolder"
+    options = {"shared": [True, False], "qt": [True, False], "mpi": [True, False],
+                "fPIC": [True, False], "minimal": [True, False], "ioxml": [True, False],
+                "ioexport": [True, False], "mpi_minimal": [True, False]}
+    default_options = {"shared": False, "qt": False, "mpi": False, "fPIC": False,
+                "minimal": False, "ioxml": False, "ioexport": False, "mpi_minimal": False}
+    topics = ("conan", "VTK", "3D rendering", "2D plotting", "3D interaction", "3D manipulation", 
+                "graphics", "image processing", "scientific visualization", "geometry modeling")
+    short_paths = True
+
+    version_split = version.split('.')
+    short_version = "{}.{}".format(version_split[0], version_split[1])
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("{}-{}".format(self.name, self.version), self.source_subfolder)
+
+    def requirements(self):
+        if self.options.qt:
+            self.requires("qt/5.15.1@bincrafters/stable")
```

Comment:
Only cci recipes are accepted.
There is a qt pr pending (see https://github.com/conan-io/conan-center-index/pull/1759), but that will not be ready soon.
In the mean time, you can add a FIXME and raise a `ConanInvalidConfiguration`.

Reply:
Done